src/GitLabApiClient/Internal/Http/GitLabApiPagedRequestor.cs

```diff
@@ -15,6 +16,31 @@ internal sealed class GitLabApiPagedRequestor
 
         public GitLabApiPagedRequestor(GitLabApiRequestor requestor) => _requestor = requestor;
 
+        public async Task<(RateLimitPagedInfo rateLimitInfo, IList<T>)> GetRateLimitPagedList<T>(string url, PageOptions pageOptions)
+        {
+            var response = await _requestor.GetWithHeaders<IList<T>>(
+                GetPagedUrl(url, pageOptions.Page, pageOptions.ItemsPerPage));
+
+            var results = response.Item1;
+            var headers = response.Item2;
```

Comment:
Cleaner to do: ` var (results, headers) = ...`

Reply:
Updated